keps/sig-scheduling/4133-priority-class-value-mutate/README.md

```diff
@@ -0,0 +1,900 @@
+-->
+
+[kubernetes.io]: https://kubernetes.io/
+[kubernetes/enhancements]: https://git.k8s.io/enhancements
+[kubernetes/kubernetes]: https://git.k8s.io/kubernetes
+[kubernetes/website]: https://git.k8s.io/website
+
+## Summary
+
+At present, we cannot update the priorityClass value directly. This enhancement proposes
+to relax update validation on priorityClass and make priorityClass value to be mutable.
+
+<!--
+This section is incredibly important for producing high-quality, user-focused
+documentation such as release notes or a development roadmap. It should be
+possible to collect this information before implementation begins, in order to
+avoid requiring implementors to split their attention between writing release
+notes and implementing the feature itself. KEP editors and SIG Docs
+should help to ensure that the tone and content of the `Summary` section is
+useful for a wide audience.
+
+A good summary is probably at least a paragraph in length.
+
+Both in this section and below, follow the guidelines of the [documentation
+style guide]. In particular, wrap lines to a reasonable length, to make it
+easier for reviewers to cite specific portions, and to minimize diff churn on
+updates.
+
+[documentation style guide]: https://github.com/kubernetes/community/blob/master/contributors/guide/style-guide.md
+-->
+
+## Motivation
+
+We'd like priority classes value to be mutable going from release 1.29 because of we're technically achieving the
+same effect with re-creation. Reasons for making priority classes immutable are mentioned in this
+[proposal](https://github.com/kubernetes/design-proposals-archive/blob/main/scheduling/pod-priority-api.md#drawbacks-of-changing-priority-classes).
+But we noticed that people are anyways deleting/creating priority classes despite the reasons mentioned above.
+While the deletion and re-creation seems like lesser of both the evils, our experience says that the deletion
+and re-creation of priority classes is more dangerous and error prone, considering people might delete another
+priority class, instead of doing just an update where an update can be rejected if there is a name mismatch or
+value update.
+
+<!--
+This section is for explicitly listing the motivation, goals, and non-goals of
+this KEP.  Describe why the change is important and the benefits to users. The
+motivation section can optionally provide links to [experience reports] to
+demonstrate the interest in a KEP within the wider Kubernetes community.
+
+[experience reports]: https://github.com/golang/go/wiki/ExperienceReports
+-->
+
+### Goals
+
+- Allow mutating priorityClass value
+
+<!--
+List the specific goals of the KEP. What is it trying to achieve? How will we
+know that this has succeeded?
+-->
+
+### Non-Goals
+
+- Allow mutating priorityClass other immutable filed like `PreemptionPolicy`
+- Change the priority of running or pending pods when changing the priority value of a priority class
```

Comment:
Can you also sort out the risks to running pods with the changed priority. Maybe at the `### Risks and Mitigations`